configs/test/bot/setup/android.bash

```diff
@@ -98,7 +110,10 @@ rm -rf clusterfuzz
 $GSUTIL_PATH/gsutil cp gs://$DEPLOYMENT_BUCKET/linux.zip clusterfuzz-source.zip
 unzip -q clusterfuzz-source.zip
 
-echo "Running ClusterFuzz."
-OS_OVERRIDE="ANDROID" ANDROID_SERIAL="$ANDROID_SERIAL" PATH="$PATH" NFS_ROOT="$NFS_ROOT" GOOGLE_APPLICATION_CREDENTIALS="$GOOGLE_APPLICATION_CREDENTIALS" ROOT_DIR="$ROOT_DIR" PYTHONPATH="$PYTHONPATH" GSUTIL_PATH="$GSUTIL_PATH" python $ROOT_DIR/src/python/bot/startup/run.py &
-
-echo "Success!"
+if [ -z "$ANDROID_SERIAL" ]; then
+  for serial in `adb devices -l | awk -F' ' '{ print $3 }' | grep usb:`; do
+    run_bot $serial &
+  done
+else
+  run_bot $ANDROID_SERIAL
```

Comment:
missing & at end ?

Reply:
Good point. The script behavior should be consistent in both cases, even though it doesn't need to run in parallel here.